### PR TITLE
PP-13135: Modularise simplified account routes

### PR DESCRIPTION
**Pull request by oswaldquek**
The obvious benefit is to make managing routes easier.

### CODE REVIEW

File: app/simplifiedAccountRoutes.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+'use strict'
```
Comment (nlsteers) [inline app/simplifiedAccountRoutes.js:1]: ES6 made this redundant in 2015, I think we can stop adding it :D

File: app/simplifiedAccountRoutes.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
```
Comment (nlsteers) [inline app/simplifiedAccountRoutes.js:2]: we use `kebab-case` for file naming in the admintool

File: app/simplifiedAccountRoutes.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+simplifiedAccount.get(paths.simplifiedAccount.settings.index, serviceSettingsController.index.get)
+
+// service name
+simplifiedAccount.get(paths.simplifiedAccount.settings.serviceName.index, permission('service-name:update'), serviceSettingsController.serviceName.get)
+simplifiedAccount.get(paths.simplifiedAccount.settings.serviceName.edit, permission('service-name:update'), serviceSettingsController.serviceName.getEditServiceName)
+simplifiedAccount.post(paths.simplifiedAccount.settings.serviceName.removeCy, permission('service-name:update'), serviceSettingsController.serviceName.postRemoveWelshServiceName)
+simplifiedAccount.post(paths.simplifiedAccount.settings.serviceName.edit, permission('service-name:update'), serviceSettingsController.serviceName.postEditServiceName)
+
+// email notifications
+simplifiedAccount.get(paths.simplifiedAccount.settings.emailNotifications.index, permission('email-notification-template:read'), serviceSettingsController.emailNotifications.get)
```
Comment (nlsteers) [inline app/simplifiedAccountRoutes.js]: pending some work to tidy up permissions in general, we're using `transactions:read` as a minimum permission level to view read-only settings

File: app/simplifiedAccountRoutes.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+const { Router } = require('express')
+
+const getSimplifiedAccount = require('./middleware/simplified-account/simplified-account-strategy.middleware')
+const isOptedInToSimplifiedAccounts = require('./middleware/simplified-account/simplified-account-opt-in.middleware')
+const userIsAuthorised = require('./middleware/user-is-authorised')
+const paths = require('./paths')
+const serviceSettingsController = require('./controllers/simplified-account/settings')
+const permission = require('./middleware/permission')
+
+const simplifiedAccount = new Router({ mergeParams: true })
```
Comment (nlsteers) [inline app/simplifiedAccountRoutes.js:12]: could just make this the default export and remove the `exports` declaration at the end of the file

**Review by nlsteers — COMMENTED** [review]
I think moving this stuff out of the routes file whilst we're working on it is not a bad idea, 💭 in the review

File: app/simplifiedAccountRoutes.js (hunk 4 above)
Comment (oswaldquek) [reply app/simplifiedAccountRoutes.js:12]: hmm not sure how to make this work, can you give a code example?

**Pull request merged**